setup.cfg

```diff
@@ -32,7 +32,7 @@ docs =
     sphinx_rtd_theme
     sphinxcontrib-bibtex
 report =
-    genno[compat]
+    genno[compat,graphviz,plotnine]
```

Comment:
`genno[compat]` is a superset of `genno[plotnine]`; see https://github.com/khaeru/genno/blob/main/setup.cfg#L44-L66. So this could be just `genno[compat,graphviz]`.